Expensify/App
# Improving autofill background appearance on password inputs

### Details
When auto filling any normal field we should see the whole input styled by autofill stylings
When autofilling a password input we should see part of the input styled with a strait border and a space before the visibility toggle button.

### Fixed Issues

$ https://github.com/Expensify/App/issues/7016

### Tests

1. At login screen auto fill email box
2. see the background fills the whole box
3. autofill the password box
4. see that the box is filled like the screenshots


### QA Steps



### Tested On


### Screenshots


#### Web
![image](https://user-images.githubusercontent.com/47184118/148696055-2e9b11d9-aabc-417d-a7dd-29182a03f450.png)

![image](https://user-images.githubusercontent.com/47184118/148696070-a4ec8f3e-64c1-4e6c-b08e-445a5804f9dd.png)

![image](https://user-images.githubusercontent.com/47184118/148696247-24182785-a7b5-41f6-bead-060f5c02f21b.png)
Hasn't regressed on other non secure inputs
#### Mobile Web
![image](https://user-images.githubusercontent.com/47184118/148840199-949d9a89-3604-4711-a023-902dd881f175.png)
![image](https://user-images.githubusercontent.com/47184118/148840296-55740e4b-031d-4c96-a1f0-1c11452945b9.png)

#### Desktop
There isn't any autofill on this

#### iOS
iOS doesn't colour backrounds on autofill. N/A

#### Android
![image](https://user-images.githubusercontent.com/47184118/148842404-ae836475-bcfc-4bb0-8d46-a2c234b12339.png)

Fix border underling for inputs on android:
![Screenshot_20220119-160937_New Expensify](https://user-images.githubusercontent.com/47184118/150183284-640f418f-84f2-479f-b087-76db383caec2.jpg)

![Screenshot_20220119-160944_New Expensify](https://user-images.githubusercontent.com/47184118/150183361-fb63f483-7ac9-4338-a6fb-d6a3638db7b0.jpg)
